rskj-core/src/test/java/co/rsk/peg/BridgeSupportProcessFundsMigrationTest.java

```diff
@@ -39,81 +40,78 @@ private static Stream<Arguments> processFundMigrationArgsProvider() {
+            Arguments.of(bridgeTestNetConstants, wasabiActivations, true),
+            Arguments.of(bridgeMainNetConstants, wasabiActivations, false),
+            Arguments.of(bridgeMainNetConstants, wasabiActivations, true)
         );
 
-        ActivationConfig.ForBlock activationsAfterRSKIP146 = mock(ActivationConfig.ForBlock.class);
-        when(activationsAfterRSKIP146.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
+        ActivationConfig.ForBlock papyrusActivations = ActivationConfigsForTest.papyrus200().forBlock(0);;
 
-        Stream<Arguments> afterRskip146Tests = Stream.of(
-            Arguments.of(bridgeTestNetConstants, activationsAfterRSKIP146, false),
-            Arguments.of(bridgeTestNetConstants, activationsAfterRSKIP146, true),
-            Arguments.of(bridgeMainNetConstants, activationsAfterRSKIP146, false),
-            Arguments.of(bridgeMainNetConstants, activationsAfterRSKIP146, true)
+        Stream<Arguments> papyrusTests = Stream.of(
+            Arguments.of(bridgeTestNetConstants, papyrusActivations, false),
+            Arguments.of(bridgeTestNetConstants, papyrusActivations, true),
+            Arguments.of(bridgeMainNetConstants, papyrusActivations, false),
+            Arguments.of(bridgeMainNetConstants, papyrusActivations, true)
         );
 
-        ActivationConfig.ForBlock activationsAfterRSKIP357 = mock(ActivationConfig.ForBlock.class);
-        when(activationsAfterRSKIP357.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
-        when(activationsAfterRSKIP357.isActive(ConsensusRule.RSKIP357)).thenReturn(true);
+        ActivationConfig.ForBlock irisActivations = ActivationConfigsForTest.iris300().forBlock(0);;
 
-        Stream<Arguments> afterRskip357Tests = Stream.of(
-            Arguments.of(bridgeTestNetConstants, activationsAfterRSKIP357, false),
-            Arguments.of(bridgeTestNetConstants, activationsAfterRSKIP357, true),
-            Arguments.of(bridgeMainNetConstants, activationsAfterRSKIP357, false),
-            Arguments.of(bridgeMainNetConstants, activationsAfterRSKIP357, true)
+        Stream<Arguments> irisTests = Stream.of(
+            Arguments.of(bridgeTestNetConstants, irisActivations, false),
+            Arguments.of(bridgeTestNetConstants, irisActivations, true),
+            Arguments.of(bridgeMainNetConstants, irisActivations, false),
+            Arguments.of(bridgeMainNetConstants, irisActivations, true)
         );
 
-        ActivationConfig.ForBlock activationsAfterRSKIP374 = mock(ActivationConfig.ForBlock.class);
-        when(activationsAfterRSKIP374.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
-        when(activationsAfterRSKIP374.isActive(ConsensusRule.RSKIP357)).thenReturn(true);
-        when(activationsAfterRSKIP374.isActive(ConsensusRule.RSKIP374)).thenReturn(true);
+        ActivationConfig.ForBlock hopActivations = ActivationConfigsForTest.iris300().forBlock(0);;
```

Comment:
Shouldn't it be hop? I wonder how these tests run if that's the case

Reply:
Fixed.